fenics/environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+- conda-forge
+dependencies:
+  - fenics =2019.1.0
+  - mshr =2019.1.0
+  - matplotlib
+  - seaborn
```

Comment:
@liamhuber  Is `seaborn` required for the fenics notebooks? I am open to include `seaborn` as part of `pyiron/base` if you say it is always useful and worth increasing the size of the image.

Reply:
@jan-janssen `matplotlib` is ~10 MB and `seaborn` is only ~300 KB -- please keep it. I'm not absolutely certain it's necessary, but it's always pleasant to have around and I don't want to make sure it's *not* used.

Reply:
@liamhuber ok

@muh-hassani Can you add `seaborn` to `pyiron/base` ?